roles/install_ansible_rulebook/tasks/install_fedora.yml

```diff
@@ -32,15 +32,15 @@
   ansible.builtin.pip:
     name:
       - pip
+      - wheel
     state: latest
 
-- name: Install pip dependencies
+- name: Ensure ansible dependencies
   ansible.builtin.pip:
     name:
       - ansible
       - ansible-runner
-      - wheel
-    state: latest
+    state: present
 
 - name: install ansible-rulebook
```

Comment:
```suggestion
- name: Install ansible-rulebook
```

Reply:
Can you capitalize the name on L53 as well?

Reply:
Can you create a PR to fix all of those issues?